accumulo/src/test/java/org/vertexium/accumulo/AccumuloResource.java

```diff
@@ -0,0 +1,148 @@
+package org.vertexium.accumulo;
+
+import org.apache.accumulo.core.client.AccumuloException;
+import org.apache.accumulo.core.client.AccumuloSecurityException;
+import org.apache.accumulo.core.client.Connector;
+import org.apache.accumulo.minicluster.MiniAccumuloCluster;
+import org.apache.accumulo.minicluster.MiniAccumuloConfig;
+import org.junit.rules.ExternalResource;
+import org.vertexium.GraphConfiguration;
+import org.vertexium.test.GraphTestBase;
+import org.vertexium.util.VertexiumLogger;
+import org.vertexium.util.VertexiumLoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.vertexium.test.GraphTestBase.*;
+
+public class AccumuloResource extends ExternalResource {
```

Comment:
Not really a ding on this PR but more of a discussion point: With this new `AccumuloResource` strategy could we break up the way to long `GraphTestBase` class?

CC @diegogrz